src/main.rs

```diff
@@ -185,6 +187,36 @@ fn select_ascii(small: bool) -> Option<Text<'static>> {
     }
 }
 
+fn list_themes() {
+    let themes = Themes::variants();
+    themes
+        .iter()
+        .for_each(|x| println!("• {} (Built-in)", x.bright_green()));
+
+    if let Some(dir) = dirs::data_local_dir() {
+        let entries = libmacchina::extra::list_dir_entries(&dir.join("macchina/themes"));
+        if !entries.is_empty() {
+            let custom_themes = entries
+                .iter()
+                .filter(|&x| {
+                    if let Some(ext) = libmacchina::extra::path_extension(&x) {
+                        return ext == "json";
+                    }
+
+                    return false;
+                })
+                .into_iter();
```

Comment:
This is a useless conversion and we don't need `into_iter()`.

Reply:
You're absolutely right :)